cnxepub/collation.py

```diff
@@ -33,7 +33,8 @@ def easybake(ruleset, in_html, out_html):
     with respective read and write ability.
 
     """
-    html = etree.parse(in_html)
+    html_parser = etree.HTMLParser(encoding="utf-8")
```

Comment:
ah, easybake uses the HTML parser because the CSS would need to use the XHTML namespaces (via `@namespace 'http://www.w3.org/1999/xhtml`;` at the top of the CSS). Gotcha